src/styles/Home.module.css

```diff
@@ -0,0 +1,276 @@
+.main {
```

Comment:
The whole idea of using stitches was to use the tools off the styled components and not use css modules?